src/Client/PodcastEpisodes.php

```diff
@@ -100,8 +100,7 @@ public function slice(int $offset, int $length = null) : Sequence
                     if (isset($this->episodes[$episode['number']])) {
                         $episodes[] = $this->episodes[$episode['number']]->wait();
                     } else {
-                        $episodes[] = $episode = $this->denormalizer->denormalize($episode, PodcastEpisode::class,
-                            null, ['snippet' => true]);
+                        $episodes[] = $episode = $this->denormalizer->denormalize($episode, PodcastEpisode::class, null, ['snippet' => true]);
```

Comment:
not sure how it was working before, but in a list PodcastEpisodes are snippets; and without it tests explode